### PR TITLE
Release v0.12.0

### LINKED ISSUE #19: Announce math release
Bringing native *math* support, the next SILE release is likely to get a little more publicity than usual. As such it might be nice to prepare for it just a little more than usual.

1. [x] An introductory paragraph for release announcement on the blog that tells a little bit about the approach for folks that have been sticking to TeX just for the math.
1. [x] An new example or two to put up as an image showcasing something it can do.

@OlivierNicole Are either of these (or anything else that you suggest should go into the release) be something you can help with?

### PR DESCRIPTION
**Pull request by alerque**
Closes #19.

- [x] Add Summary of improvements
- [x] Add new example to showcase math
- [x] Add release changelog
- [x] Post release manual

### CODE REVIEW

**Comment by alerque** [comment]
@OlivierNicole Feel free to add review suggestions here. You are more familiar with how it actually works, I just composed this based on my observations from commit messages and the manual. If there is anything we should convey that would be useful to set expectations for people interested in Math coming to this for the first time just jump in and jot them down.

File: _drafts/SILE-0.12.0-is-released.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+post_gravatar: 11cdaff4c6f9b290db40f69d3b20caf1
+---
+Version v0.12.0 of SILE has been released and is [available for download][release]!
+See the included [CHANGELOG.md][changelog] or review the [commit history][commits] for more explicit details.
+
+### Summary of Improvements
+
+Please welcome *MATH* typesetting!
+
+While this release follows close on the heals of previous releases it represents over 2 years of work from community contributors.
```
Comment (simoncozens) [inline _drafts/SILE-0.12.0-is-released.md]: heels

File: _drafts/SILE-0.12.0-is-released.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+Rui Xia started work on a native math feature in April of 2019.
+While this initial phase was never completed, it did get as far as a working prototype.
+In November of 2020, Oliver Nicole picked up on the work and transformed the prototype into the shape as found in this release.
+Additional thank to Mateusz Czapliński for input throughout the process.
```
Comment (simoncozens) [inline _drafts/SILE-0.12.0-is-released.md]: thanks

File: _drafts/SILE-0.12.0-is-released.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+On a conceptual level the math support is based on [MathML][] and relies on OpenType math fonts.
+The input notation may be MathML (either directly using the native XML syntax or using an 1-to-1 equivalent SIL syntax) or an alternative math shorthand syntax similar to the math syntax used by TeX.
```
Comment (simoncozens) [inline _drafts/SILE-0.12.0-is-released.md]: "SILE syntax". But there are too many syntaxes in that sentence.

> The input for math typesetting can either be in MathML format (expressed using the native XML format or using equivalent SILE commands) or in an alternative math shorthand syntax, similar to that of TeX.

File: _drafts/SILE-0.12.0-is-released.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+See section 5.29.5 in the [v0.12.0 manual][manual] for detailed documentation on the new math package.
+Examples are given for various formulas and syntax options.
+A math example has also been added to the [packages example gallery](/examples/packages.html) on this website.
+
+Additionally Didier Willis contributed several new features and fixes including much more control over Table of Contents, styling options for PDF links, correct French punctuation treatment, and more.
+
+Thanks to the contributions of Doron Behar, as of this release SILE is usable as a [Nix Flake][flake] (in addition to the usual Nix package).
+This is an easy way to test run the latest Git HEAD version at any time from almost any OS.
+One you have flake support enabled, just run `nix run github:sile-typesetter/sile -- <sile orguments>`.
+This website use this to regenerate some of the examples in the gallery.
```
Comment (simoncozens) [inline _drafts/SILE-0.12.0-is-released.md]: uses

File: _drafts/SILE-0.12.0-is-released.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+As major a new feature as the math package is, at the end of the day the major version bump is on account of a breaking change elsewhere.
```
Comment (simoncozens) [inline _drafts/SILE-0.12.0-is-released.md]: > As major a new feature as the math package is, at the end of the day

This is awkward. Consider:

"Even though the math package is a major new feature, ultimately the major version bump was caused by a breaking change elsewhere."

File: _drafts/SILE-0.12.0-is-released.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+Previous to this release, the font used for footnotes was the SILE default unless specifically set.
+And setting changes were relative to the SILE default.
+This behaviour was workable, but usually came as a surprise to most users.
+With this release, footnote font setting default to being relative to the current document font.
```
Comment (simoncozens) [inline _drafts/SILE-0.12.0-is-released.md]: > footnote font setting default

the footnote font setting defaults

**Review by simoncozens — COMMENTED** [review]
Just a few grammar suggestions. But otherwise, thank you for a thorough and important update.

**Pull request merged**